MuscleModel/ReadFromModel/settings_Muscle_analysis_v4.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
why did you add this file ?

Reply:
I put it there in case version 4 of OpenSim had a different implementation. This file can be removed.